build.gradle

```diff
@@ -120,6 +120,21 @@ allprojects {
       content { includeGroupByRegex('com\\.splunk\\..*') }
     }
     mavenCentral()
+
+    // ethereum execution spec tests fixtures. Exclusively for ethereum submodule to run ref tests
+    def ethExecSpecTestsRepo = ivy {
+      url 'https://github.com'
```

Comment:
`ivy` repo definition needs a url (or local path).

Reply:
So why can't we use `url 'https://github.com/ethereum/execution-spec-tests'` and drop the filter?

Reply:
because of this: `referenceTestImplementation 'ethereum:execution-spec-tests:0.2.3:fixtures@tar.gz'`. We are taking advantage of organization, module, version since thats how the dependencies are declared and using them as variables to be used in the custom ivy module.

Reply:
I've added additional comment in referencetest build.gradle as well.